src/components/TableFields.js

```diff
@@ -6,15 +6,13 @@ import './TableFields.css';
 

```

Comment:
Maybe to revert initial array to original and here to create array:
`const pieceImages={
P:'♙',R:'♖',N:'♘',B:'♗',Q:'♕',K:'♔',
p:'♟',r:'♜',n:'♞',b:'♝',q:'♛',k:'♚',
}`
and to outuput with: `{pieceImages[chessField.piece]}`